src/main/java/medic/gateway/alert/SmsSender.java

```diff
@@ -114,22 +119,23 @@ private void sendSms_dummy(WoMessage m) {
 		db.updateStatus(m, SENT, DELIVERED);
 	}
 
-	private ArrayList<PendingIntent> intentsFor(String intentType, WoMessage m, ArrayList<String> parts) {
+	private ArrayList<PendingIntent> intentsFor(String intentType, WoMessage m, int mIdx, ArrayList<String> parts, Class<?> receiverClass) {
```

Comment:
Why pass in the `receiverClass` when it's always `IntentProcessor.class`? Can't we just hardcode that in the implementation?

Reply:
Done!